dartsim/src/SDFFeatures_TEST.cc

```diff
@@ -205,14 +399,15 @@ TEST(SDFFeatures_TEST, CheckDartsimData)
 
 /////////////////////////////////////////////////
 // Test that joint limits are working by running the simulation
-TEST(SDFFeatures_TEST, CheckJointLimitEnforcement)
+TEST_P(SDFFeatures_TEST, CheckJointLimitEnforcement)
 {
-  World world = LoadWorld(TEST_WORLD_DIR"/test.world");
+  WorldPtr world = LoadWorld(TEST_WORLD_DIR"/test.world");
```

Comment:
I am seeing `this->GetParam()` used as a parameter for the `LoadWorld` call in some of the tests, but not all of them - if I understand correctly, `this->GetParam()` will load the world both as a whole and piecemeal instead of loading as a whole only, right? If so, what is the criteria for determining if a test needs `this->GetParam()` in the `LoadWorld` call or not?

Reply:
I only added that on tests that I thought might fail with `LoadWorldPieceMeal`, but now that you mentioned it, I changed all of them to use both loading methods. https://github.com/ignitionrobotics/ign-physics/pull/220/commits/53ce2b79c4059185820f7d5ded29221a9cabcb2b